### PR TITLE
Adds a Stone sword (Please don't make minecraft references or i will explode you)

### PR DESCRIPTION
**Pull request by Guidesu**
## About The Pull Request
Pretty simple, adds a stone sword. I think it is balanced.
Can easily check yourself if needed
image
This is the sprite

### CODE REVIEW

File: code/game/objects/items/rogueweapons/melee/swords.dm [hunk 1]
```diff
@@ -82,6 +82,22 @@
 	swingdelay = 0
 	item_d_type = "stab"
 
+/obj/item/rogueweapon/sword/stone
+	force = 15 //Weaker than a short sword
+	force_wielded = 20 // Less than other swords
```
Comment (kittysmooch) [inline code/game/objects/items/rogueweapons/melee/swords.dm:87]: this makes it stronger than the short sword for most characters, and stronger than most iron and steel knives and daggers. a force of 15 and no wieldability is fine.

File: code/game/objects/items/rogueweapons/melee/swords.dm [hunk 2]
```diff
@@ -82,6 +82,22 @@
...
+	possible_item_intents = list(/datum/intent/sword/cut, /datum/intent/sword/thrust)
+	gripped_intents = list(/datum/intent/sword/cut, /datum/intent/sword/thrust, /datum/intent/sword/chop)
+	name = "stone sword"
+	desc = "A simple stone sword, crude and effective."
+	icon_state = "stone_sword"
+	max_blade_int = 100
+	max_integrity = 125
```
Comment (kittysmooch) [inline code/game/objects/items/rogueweapons/melee/swords.dm:94]: this also gives it functionally the same durability as iron swords. the stone knife has 50 in each by comparison.

File: code/game/objects/items/rogueweapons/melee/swords.dm [hunk 3]
```diff
@@ -82,6 +82,22 @@
...
+	anvilrepair = /datum/skill/craft/crafting
+	smeltresult = /obj/item/natural/stone
```
Comment (kittysmooch) [inline code/game/objects/items/rogueweapons/melee/swords.dm:96]: i don't think this needs to be smeltable at all

**Review by kittysmooch — COMMENTED** [review]
this is a bit strong for something you don't need any infrastructure to make

**Comment by Lutowski** [comment]
Should have significantly lower integrity since you can repair it.

**Pull request merged**